bin/post_compile

```diff
@@ -80,21 +97,23 @@ heroku-install-cabot-plugins() {
   done
 }
 
+heroku-put-scripts-on-path() {
+  # A better way of doing this perhaps?
+  ln -sf /app/.heroku/vendor/node-4.2.6/bin/* /app/.heroku/python/bin/
+}
+
 heroku-compress-assets() {
   puts-step "Compressing assets"
-  PATH=$PATH:/app/$NODE_PATH/bin .heroku/python/bin/python manage.py compress | indent
+  PATH=$PATH:/app/$NODE_PATH/bin .heroku/python/bin/python manage.py compress --force | indent
 }
 
 
 main() {
-  if [[ ! -n $DYNO ]]; then
```

Comment:
Ah, ok, fair enough. I suppose the alternative is to detect a buildpack environment some other way. A (non-idea) option would be to test for the existence of `/app`.

Alternatively, this entire `post_compile` script could actually be moved out of cabot and into an actual buildpack. One could then specify the cabot buildpack for their heroku/flynn app, which would do all the installing and config for them. The user could have the option of providing their own custom config files in the root directory if they desired.